tiny-event-sourcing-spring-app/src/main/kotlin/ru.quipy/demo/UserAggregate.kt

```diff
@@ -0,0 +1,98 @@
+package ru.quipy.demo
+
+import ru.quipy.core.AggregateType
+import ru.quipy.domain.Aggregate
+import java.util.*
+
+@AggregateType(aggregateEventsTableName = "aggregate-user")
+data class UserAggregate(
+    override val aggregateId: String
+) : Aggregate {
+    override var createdAt: Long = System.currentTimeMillis()
+    override var updatedAt: Long = System.currentTimeMillis()
+
+    var userName: String = ""
+    var userLogin: String = ""
+    var userPassword: String = ""
+    lateinit var defaultPaymentId: UUID
```

Comment:
why some lateinit var some not?

Reply:
Because some are passed in constructor, and some can be initialized a long time after maybe never if user never chooses defaultpayment.

Reply:
Do everything late init